### PR TITLE
#636_Address lodash vulnerability. Bug fix.

### PR DESCRIPTION
**Pull request by HenkKodde**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,8 +27,8 @@
   },
   "dependencies": {
     "bluebird": "3.5.0",
-    "knex": "0.14.2",
-    "lodash": "4.17.5"
+    "knex": "^0.16.3",
+    "lodash": "^4.17.11"
```
Comment (rmothilal) [inline package.json]: Please remove the carets

File: package.json [hunk 2]
```diff
@@ -37,7 +40,7 @@
     "proxyquire": "1.8.0",
     "sinon": "4.2.2",
     "standard": "10.0.3",
-    "tap-xunit": "2.2.0",
+    "tap-xunit": "^2.3.0",
```
Comment (rmothilal) [inline package.json]: Please remove the caret

File: package.json (hunk 1 above)
Comment (HenkKodde) [reply package.json]: Have removed the carets as requested.

File: package.json (hunk 2 above)
Comment (HenkKodde) [reply package.json]: carets removed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
 # central-services-database
+[![Git Commit](https://img.shields.io/github/last-commit/mojaloop/central-services-database.svg?style=flat)](https://github.com/mojaloop/central-services-database/commits/master)
+[![Git Releases](https://img.shields.io/github/release/mojaloop/central-services-database.svg?style=flat)](https://github.com/mojaloop/central-services-database/releases)
+[![CircleCI](https://circleci.com/gh/mojaloop/central-services-database.svg?style=svg)](https://circleci.com/gh/mojaloop/central-services-database)
```
Comment (mdebarros) [inline README.md]: Please add the remaining badges.

Here is an example:
```MARKDOWN
[![Git Commit](https://img.shields.io/github/last-commit/mojaloop/central-services-metrics.svg?style=flat)](https://github.com/mojaloop/central-services-metrics/commits/master)
[![Git Releases](https://img.shields.io/github/release/mojaloop/central-services-metrics.svg?style=flat)](https://github.com/mojaloop/central-services-metrics/releases)
[![Npm Version](https://img.shields.io/npm/v/@mojaloop/central-services-metrics.svg?style=flat)](https://www.npmjs.com/package/@mojaloop/central-services-metrics)
[![NPM Vulnerabilities](https://img.shields.io/snyk/vulnerabilities/npm/@mojaloop/central-services-metrics.svg?style=flat)](https://www.npmjs.com/package/@mojaloop/central-services-metrics)
[![CircleCI](https://circleci.com/gh/mojaloop/central-services-metrics.svg?style=svg)](https://circleci.com/gh/mojaloop/central-services-metrics)
```

**Pull request merged**